pvm/ts/src/engine/texasHoldem-action-index.test.ts

```
@@ -21,101 +21,101 @@ describe("Texas Holdem - Turn Index", () => {
 
-    describe("Turn Index Initialization", () => {
-        it.only("should initialize with turn index of 0", () => {
+    describe("Action Index Initialization", () => {
+        it.only("should initialize with turn index of 1", () => {
             // Check initial turn index is 0
-            expect(game.getTurnIndex()).toBe(0);
+            expect(game.getActionIndex()).toBe(1);
         });
 
-        it("should reset turn index to 0 when game is reinitialized", () => {
+        it.only("should reset turn index to 0 when game is reinitialized", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)